plugin-docker/che-plugin-docker-client/src/main/java/org/eclipse/che/plugin/docker/client/DockerConnectorConfiguration.java

```diff
@@ -203,23 +264,18 @@ protected String getDockerHostIp(final boolean isLinux, @NotNull final Map<Strin
     }
 
     private static String dockerMachineCertsDirectoryPath() {
-        return dockerMachineCertsDirectoryPath(SystemInfo.isLinux(), System.getenv());
+        return dockerMachineCertsDirectoryPath(System.getenv());
     }
 
     /**
      * Provides the location of certificates used to connect on docker.
      * It may use extra environment to help to build this path
      *
-     * @param isLinux
-     *         if System is running on Linux
      * @param env
      *         should contain System environment
      * @return local path of the docker certificates
      */
-    protected static String dockerMachineCertsDirectoryPath(boolean isLinux, @NotNull Map<String, String> env) {
-        if (isLinux) {
-            return null;
-        }
+    protected static String dockerMachineCertsDirectoryPath(@NotNull Map<String, String> env) {
```

Comment:
If it is required not to be `null` may be you should add appropriate check e.g `Objects.requireNonNull(env);`